src/Locales/ar/Pode.psd1

```diff
@@ -0,0 +1,167 @@
+ConvertFrom-StringData -StringData @'
```

Comment:
Any reason these can't just be direct hashtables, without calling `ConvertFrom-StringData` - seeing as that just returns as hashtable anyways? 🤔

Reply:
The powershell documentationuses this form. I haven't try without

Reply:
The formatting I can understand, I've used translation services where they have strict formats they can only give you translations in, and this format is pretty common.

I saw the link, and then spotted `ConvertFrom-StringData` and `.psd1` were introduced in PS2, and the page has basically never been updated since then 😂 so was thinking it might be a super old-school PowerShell format for the original `.psd1`.

If it simplifies mapping from translation services and AI, I'm happy to resolve this then.

Reply:
So far, I've taught OpenAI to generate in this format for all languages. Later, I'm going to convert it to the native psd1 format.

I agree with you this looks like it's old school.  :)